index.js

```diff
@@ -0,0 +1,59 @@
+const titleInputEl = document.getElementById('title-input');
+const authorInputEl = document.getElementById('author-input');
+const formEl = document.getElementById('form');
+const collectionsSectionEl = document.getElementById('books-collection');
+
+const collectionData = JSON.parse(window.localStorage.getItem('collectionArray')) || [];
+const removeButtons = [];
+
+const renderBooks = (collectionData) => {
+  collectionsSectionEl.innerHTML = '';
+  collectionData.forEach((book, index) => {
+    const bookEntry = document.createElement('div');
+    bookEntry.classList.add('book-entry');
+    const bookTitle = document.createElement('p');
+    bookTitle.classList.add('book-title');
+    bookTitle.textContent = book.title;
+    const bookAuthor = document.createElement('p');
+    bookAuthor.classList.add('book-author');
+    bookAuthor.textContent = book.author;
+    removeButtons[index] = document.createElement('button');
+    removeButtons[index].classList.add('remove-button');
+    removeButtons[index].textContent = 'Remove';
+    removeButtons[index].dataset.id = book.bookID;
+    bookEntry.append(bookTitle, bookAuthor, removeButtons[index]);
+    collectionsSectionEl.appendChild(bookEntry);
+  });
+
+  const addBook = (book) => {
+    collectionData.push(book);
+    window.localStorage.setItem('collectionArray', JSON.stringify(collectionData));
+  };
+
+  formEl.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const title = titleInputEl.value;
+    const author = authorInputEl.value;
+    const removeIndex = collectionData.length;
+    const book = { title, author, bookID: removeIndex };
+    addBook(book);
+    renderBooks(collectionData);
+    window.location.reload();
+  });
```

Comment:
- [ ] You did great in implementing the add book functionality 👏 . However as the previous reviewer suggested, make sure that the page is not reloaded when a book is added. Kindly consider removing `window.location.reload()`